app/models/friendship.rb

```diff
@@ -0,0 +1,67 @@
+class Friendship < ActiveRecord::Base
+  extend TokenContext
+  include Tokenizer
+	enum friendship_status: [ :denied, :pending, :accepted]
+  attr_accessor :friendship_request_token
+	cattr_accessor :daily_request_limit
```

Comment:
Typo here.  I am skeptical whether you even need an accessor for daily_request_limit since it is definitely not working here and you declare it as a class variable below